libs/src/api/rewards.js

```diff
@@ -480,19 +481,23 @@ class Rewards extends Base {
 
       needsAttestations = []
       attestations.forEach(a => {
-        if (a.res.error === GetAttestationError.CHALLENGE_INCOMPLETE) {
+        if (a.res.error === GetAttestationError.CHALLENGE_INCOMPLETE ||
+          a.res.error === GetAttestationError.MISSING_CHALLENGES) {
           needsAttestations.push(a.endpoint)
           logger.info(`Node ${a.endpoint} challenge still incomplete for challenge [${challengeId}], userId: ${encodedUserId}`)
         } else {
           completedAttestations.push(a.res)
+          if (a.res.error) {
+            unrecoverableError = true
+          }
```

Comment:
Do we want to record what the error is?

Reply:
It's gonna get logged at a higher level anyways, I think we're good. Already have so much redundant logging on this flow 😆